src/content/language/collections.md

```diff
@@ -242,78 +250,180 @@ final constantMap = const {2: 'helium', 10: 'neon', 18: 'argon'};
 For more information about maps, refer to the Maps section of the
 [`dart:core` documentation](/libraries/dart-core#maps).
 
-## Operators
+[Using constructors]: /language/classes#using-constructors
+[`Map`]: {{site.dart-api}}/dart-core/Map-class.html
+[type inference]: /language/type-system#type-inference
+
+## Supplementals
 
-### Spread operators
+The following supplemental materials can be used with
```

Comment:
The language spec calls these "elements". So a list literal contains a series of elements, each of which might be an expression (i.e. `1`), a spread element (`...foo`)`, an if element, or a for element.

We don't *have* to follow the spec terminology. We don't always do a great job of coming up with names for stuff when we specify them. But we try and in this case, I think the name works OK.

It's up to you, though. :)

Reply:
I was wondering why we didn't use the term 'element' in this guide. I've updated this for `List` and `Set`, but I've kept key-value pair for `Map`. I think we should follow the spec on this. Element is a very common term. To me, it seems like an anti-pattern if we don't use it. Take a close look at my changes.